tools/storm/helpers/acr.go

```diff
@@ -0,0 +1,191 @@
+package helpers
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/microsoft/storm"
+	"github.com/sirupsen/logrus"
+)
+
+type AcrHelper struct {
+	args struct {
+		Push                  bool     `help:"If set, push images to ACR" type:"bool"`
```

Comment:
can Push and Remove be specified for a single invocation of the helper?  what would that mean?  

another option would be an Action arg that can be Push or Remove

Reply:
is push and then immediately remove a scenario we envision?  not a big deal to me, but push and remove feel like they should be mutually exclusive in a single call to this helper.